cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help

```diff
@@ -1,16 +1,17 @@
 NAME
-       Ballerina add - Add a Ballerina module in a project
+       ballerina add - Add a new Ballerina module to a project
+
 
 SYNOPSIS
-       ballerina add <module-name> [-t|--template <template-name>]
+       ballerina add <module-name> [-t | --template <template-name>]
```

Comment:
Is it just me or does this seem like

`-t` 
OR
`--template <template-name>`

Should this be 
`[-t <template-name> | --template <template-name>]` instead?

Reply:
It seems like we are not following the convention used for these options. Have a look at https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html

If an option starts with `--`, it's a long option. And for long options, you give values as `--opt=<value>`. Not `--opt <value>`. Checked in man pages of git, grep etc. and they all follow the same convention

Reply:
Ah nvm. The key=value convention seem to be used when the value is optional.